guides/common/modules/con_importing-content.adoc

```diff
@@ -9,6 +9,9 @@ ifdef::orcharhino[]
 If you want to import RPM or DEB packages, Files, or Puppet Modules to {Project}, it is done with largely the same procedures in this chapter.
 endif::[]
 
+NOTE: If you have a disconnected environment and some of your {ProjectServer}s are on versions older than 6.10 while others are on 6.10, content import and content export do not work.
```

Comment:
Please do not litter the docs with notes. If you read our docs, we are very careful about the use of notes. 
Please respect that this is a single sourced content that is used for multiple projects and products. You adding version numbers for a Red Hat product is not helpful to our upstream or other contributors.